openproblems/data/brain_10x.py

```diff
@@ -0,0 +1,25 @@
+import os
+import tempfile
+
+import numpy as np
+import anndata
+import scprep
+import scanpy as sc
+
+from .utils import loader
+
+
+@loader
+def load_10xbrain(test=False):
```

Comment:
Can we swap this for a smaller dataset for now?

Reply:
definitely. will search for one and make a (separate) PR for it per the desired format thanks